managers/bosh-manager/BOSHManager.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Promise = require('bluebird');
+const eventmesh = require('../../data-access-layer/eventmesh');
+const logger = require('../../common/logger');
+const CONST = require('../../common/constants');
+const BaseManager = require('../BaseManager');
+const DirectorService = require('./DirectorService');
+const errors = require('../../common/errors');
+const utils = require('../../common/utils');
+const ServiceInstanceNotFound = errors.ServiceInstanceNotFound;
+
+class BOSHManager extends BaseManager {
+
+  init() {
+    const queryString = `state in (${CONST.APISERVER.RESOURCE_STATE.IN_QUEUE},${CONST.APISERVER.RESOURCE_STATE.UPDATE},${CONST.APISERVER.RESOURCE_STATE.DELETE})`;
```

Comment:
Can we have this query string in utility function as getting used in all managers ?